catroid/src/main/res/values-ar/strings.xml

```diff
@@ -1447,4 +1447,14 @@
   <string name="crash_dialog_always_send">دائماً أرسل</string>
   <string name="crash_dialog_send">أرسل</string>
   <!--  -->
+
+  <!--  -->
+  <string name="Byte_short">بايت</string>
+  <string name="kiloByte_short">كيلوبايت</string>
```

Comment:
Ok, please close this pull request on github then!
I will look at the other one ;-)